modules/test/playwright/package.json

```diff
@@ -24,6 +26,8 @@
 		"checkFormat": "tsc && liferay-npm-scripts lint && liferay-npm-scripts format:check",
 		"format": "tsc && liferay-npm-scripts lint:fix && liferay-npm-scripts format",
 		"playwright": "playwright",
+		"setUpCI": "ts-node env/setUp.env.ts",
```

Comment:
How about `ci:setup` and `ci:teardown` as in my PR? We use that convention in some other places, in Liferay tooling...

Reply:
Okay, I'll make this change here: 6a0f092